src/app/WriteHandler.h

```diff
@@ -141,6 +141,20 @@ class WriteHandler : public Messaging::ExchangeDelegate
      */
     void Close();
 
+    void DeliverListWriteBegin(const ConcreteAttributePath & aPath);
+    void DeliverListWriteEnd(const ConcreteAttributePath & aPath, bool writeWasSuccessful);
+
+    // Deliver the signal that we have delivered all list entries to the AttributeAccessInterface. This function will be called
+    // after handling the last chunk of a series of write requests. Or the write handler was shutdown (usually due to transport
+    // timeout).
+    void DeliverFinalListWriteEnd(bool writeWasSuccessful);
```

Comment:
This should document that it will only do something if we never did a `DeliverFinalListWriteEndForGroupWrite` for the relevant paths...

Reply:
Done, "DeliverFinalListWriteEndForGroupWrite will make this function no-op"